src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -187,9 +187,9 @@ module Types = struct
 
       let num_accounts = int_option_entry "Global number of accounts"
 
-      let blockchain_length = int_option_entry "Block height"
+      let block_height = int_option_entry "Block Height"
```

Comment:
Can you lowercase all the non-starting letters? We've changed style between current testnet and `develop`.